event_routing_backends/processors/caliper/event_transformers/problem_interaction_events.py

```diff
@@ -72,9 +72,12 @@ def get_object(self):
         Returns:
             dict
         """
-        object_id = self.find_nested('problem_id')
-        if not object_id:
-            object_id = self.find_nested('module_id')
+        object_id = None
+        event_data = None
+        data = self.event.get('data', None)
```

Comment:
I believe if we use `{}` instead of `None` then we wouldn't need to add an `if` statement here

Reply:
in some cases value of `data` not dict in that case we need that `if` statement. Example of such event is [here](https://github.com/edx/event-routing-backends/blob/master/event_routing_backends/processors/tests/fixtures/current/problem_check(browser).json#L18).